pkgs/servers/sql/postgresql/default.nix

```
@@ -8,6 +8,8 @@ let
 
       # This is important to obtain a version of `libpq` that does not depend on systemd.
       , enableSystemd ? (lib.versionAtLeast version "9.6" && !stdenv.isDarwin)
+      , gssSupport ? with stdenv.hostPlatform; !isWindows && !isStatic, libkrb5 ? null
```

```suggestion
      , gssSupport ? with stdenv.hostPlatform; !isWindows && !isStatic, libkrb5
```

libkrb5 should never be null and overriding with null is more of an anti pattern.

---

Rebased on staging, and applied this change.

As a side note, I did copy that method from curl directly:  https://github.com/NixOS/nixpkgs/blob/dc68eb58bb8d4dc3cfb8896932e6146c45579932/pkgs/tools/networking/curl/default.nix#L11